apps/user/services/authentication.go

```diff
@@ -48,12 +54,32 @@ func (service AuthenticationService) Authorize(c *gin.Context) (
 		return nil, errors.New("invalid state")
 	}
 
-	// TODO replace with oauth logic
-	email, _ := url.QueryUnescape(c.Query("code"))
-	user := userModels.User{}
+	tok, err := googleOauthConf.Exchange(context.TODO(), c.Query("code"))
+	if err != nil {
+		logrus.Info("Error occurred while exchanging code with token")
+		return nil, err
+	}
+
+	client := googleOauthConf.Client(context.TODO(), tok)
+
+	googlePlusAPIRequester, err := plus.New(client)
```

Comment:
googlePlusAPIRequester could be named "plusService" following the google naming convention.

Reply:
done